src/components/SecondaryPanes/Expressions.js

```diff
@@ -125,10 +125,16 @@ class Expressions extends Component<Props, State> {
   };
 
   handleNewSubmit = async (e: SyntheticEvent<HTMLFormElement>) => {
+    const { inputValue } = this.state;
+    this.props.clearExpressionError();
```

Comment:
hmm, I would prefer not firing this unless we need to

Reply:
I used this because I want to be sure that i cleaned the red borders when I'm writing a correct expression, but also permit to do an animation every time I write a wrong expression (before if I wrote 2 consecutives wrong expression, the second one didn't have an animation).

Reply:
@wldcordeiro mind taking a look at this?